compiler/modules/bir/bir.bal

```diff
@@ -87,6 +87,7 @@ public type FunctionCode record {|
 # by referencing the name of the function as a variable
 # reference.
 public type FunctionSignature readonly & record {|
+    boolean isVarArg;
```

Comment:
Why do we need this?  `restParamType` tells us whether it is varargs

Reply:
fixed